### DISCUSSION TITLE
Hisense connectlife

### DISCUSSION THREAD

**Discussion by liquidand** [indomus/forum#3110] [HUB: Home Assistant]
Salve. Ho acquistato dei climatizzatori hisense connessi tramite app connectlife. Qualcuno è in grado di aiutarmi ad integrarli su home assistant?

**Comment by KC-inDomus** [comment]
https://indomus.it/formazione/home-assistant-come-capire-come-integrare-un-componente-domotico-o-meno/

**Comment by Infotainmentgb7** [comment]
> Salve. Ho acquistato dei climatizzatori hisense connessi tramite app connectlife. Qualcuno è in grado di aiutarmi ad integrarli su home assistant
Ciao alla fine sei riuscito ad integrarli perche' io sto impazzendo

**Reply by liquidand** [reply]
Ciao. Si, li ho connessi all'app connectlife e poi usato questa integrazione:
https://github.com/oyvindwe/connectlife-ha
